app/controllers/hyrax/admin/workflow_roles_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Hyrax
+  module Admin
+    class WorkflowRolesController < ApplicationController
+      before_action :require_permissions
+      layout 'admin'
+
+      def index
+        # TODO: Find a better way to make sure workflows are loaded
```

Comment:
Do we need to load them per tenant? Can we do it when a tenant is created as we do with admin accounts?